utils/sub8_ros_tools/sub8_tools/bounds.py

```diff
@@ -0,0 +1,32 @@
+#!/usr/bin/env python
+import numpy as np
+
+import rospy
+import sub8_tools
+from twisted.internet import defer
+
+from sub8_msgs.srv import Bounds, BoundsResponse
+
+
+class BoundsServer(object):
+    def __init__(self):
+        self.bounds = np.array([[5,5], [-5, 5], [-5, -5], [5, -5]])
+        self.enforce = True
+        rospy.Service('/get_bounds', Bounds, self.got_request)
+
+
+    def got_request(self, req):
+        # to_frame = "enu" if req.to_frame == '' else req.to_frame
+        # map frame?        
+        resp = BoundsResponse(enforce=False)
+
+        bounds = [sub8_tools.numpy_to_point(i) for i in self.bounds]
```

Comment:
Prefer to use mil tools now instead of sub tools for these numpy/ros conversion functions